### PR TITLE
man: osdmaptool: document test-map-pgs* options

### PR DESCRIPTION
**Pull request by tchaikov**
Signed-off-by: Kefu Chai <kchai@redhat.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
@vumrao i just updated the manpage for osdmaptool, could you help review it?

**Comment by vumrao** [comment]
@tchaikov Thanks Kefu. lgtm.

File: doc/man/8/osdmaptool.rst [hunk 1]
```diff
@@ -49,6 +49,15 @@ Options
...
    attribute will be set to numosd shifted by bitsperosd.
 
+.. option:: --test-map-pgs [--pool poolid]
+
+   will print out the mappings of all placement groups to OSDs.
+
+.. option:: --test-map-pgs-dump [--pool poolid]
+
+   will print out the summary of all placement groups and the mappings
+   of from them to the mapped OSDs.
```
Comment (tchaikov) [inline doc/man/8/osdmaptool.rst]: s/of//

File: doc/man/8/osdmaptool.rst [hunk 2]
```diff
@@ -49,6 +49,15 @@ Options
    be set with bitsperosd bits per OSD. That is, the pg_num map
    attribute will be set to numosd shifted by bitsperosd.
 
+.. option:: --test-map-pgs [--pool poolid]
+
+   will print out the mappings of all placement groups to OSDs.
```
Comment (tchaikov) [inline doc/man/8/osdmaptool.rst]: s/of/from/

**Comment by tchaikov** [comment]
changelog

- addressed my comments.

File: doc/man/8/osdmaptool.rst [hunk 3]
```diff
@@ -61,6 +70,77 @@ To view the result::
...
+    * the weight of this OSD.
+ #. Looking at the number of placement groups held by 3 OSDs. We have
+
+    * avarge, stddev, stddev/average, expected stddev, expected stddev / average
+    * min and max
+ #. The number of placement groups mapping to n OSDs. In this case, we have 8 placement
+    groups sizing 3 replicas.
+
+In a less-balanced cluster, we could have following output for the statistics of
+placement group distribution, whose std deviation is 1.41421::
```
Comment (xiexingguo) [inline doc/man/8/osdmaptool.rst]: @tchaikov  You might want to change  std to standard, which looks more formal and better.

**Comment by xiexingguo** [comment]
lgtm also

**Comment by tchaikov** [comment]
changelog:

- s/std deviation/standard deviation/

**Pull request merged**